frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -447,142 +465,226 @@ export function Experiment(): JSX.Element {
+                                                    <b>{variant.key}</b>
+                                                </Tag>
+                                            </div>
+                                        </Col>
+                                    ))}
+                                <Col className="ml">
+                                    <div className="card-secondary">Control conversion rate</div>
+                                    <div className="text-center">
+                                        <b>--</b>{' '}
+                                        <Tag style={{ border: 'none', color: 'white' }} color={resultsTagColors[3]}>
+                                            <b>Control</b>
+                                        </Tag>
+                                    </div>
+                                </Col>
+                            </Row>
+                        )}
+                        {experimentResults && !experimentResults?.noData ? (
+                            <BindLogic
+                                logic={insightLogic}
+                                props={{
+                                    dashboardItemId: experimentResults.itemID,
+                                    filters: {
+                                        ...experimentResults.filters,
+                                        insight: 'FUNNELS',
+                                        funnel_viz_type: FunnelVizType.Steps,
+                                        display: 'FunnelViz',
+                                        layout: FunnelLayout.vertical,
+                                    },
+                                    cachedResults: experimentResults.insight,
+                                    syncWithUrl: false,
+                                    doNotLoad: true,
+                                }}
+                            >
+                                <div className="mt">
+                                    <InsightContainer disableTable={true} />
+                                </div>
+                            </BindLogic>
+                        ) : (
+                            <div
+                                style={{
+                                    display: 'flex',
+                                    justifyContent: 'center',
+                                    alignItems: 'center',
+                                    marginTop: 16,
+                                    background: '#FAFAF9',
+                                    border: '1px solid var(--border)',
+                                    width: '100%',
+                                    minHeight: 320,
+                                    fontSize: 24,
+                                }}
+                            >
+                                <b>There are no results for this experiment yet.</b>
```

Comment:
Note for self: revisit this after I optimise result calculation speed. (we might need a loading state).

Right now, I have to wait 10s with an empty screen, waiting for the request to complete, during which it says "no data". Sigh. Not great.